### PR TITLE
CT: Mobile "I'll be taking classes outside of the U.S. and U.S. territories" not selectable #9872

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
https://github.com/department-of-veterans-affairs/va.gov-team/issues/9872

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx [hunk 1]
```diff
@@ -187,7 +187,10 @@ class EstimateYourBenefitsForm extends React.Component {
 
   handleInternationalCheckboxFocus = e => {
     if (window.innerWidth <= SMALL_SCREEN_WIDTH) {
-      e.target.scrollIntoView();
+      const field = document.getElementById('classesOutsideUS');
+      if (field) {
```
Comment (shawkey-daniel) [inline src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx:191]: Is this a place for that optional chaining operator?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx (hunk 1 above)
Comment (zurbergram) [reply src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx:191]: doing `document.getElementById('classesOutsideUS')?.scrollIntoView();` gets `ESLint: Expected an assignment or function call and instead saw an expression.(no-unused-expressions)` error message when trying to commit

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx [hunk 2]
```diff
@@ -187,7 +191,10 @@ class EstimateYourBenefitsForm extends React.Component {
...
+        scrollAndFocus(field);
```
Comment (zurbergram) [inline src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx:196]: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/utilities/ui/index.js#L60

**Review by sonntag-adam — APPROVED** [review]

**Pull request merged**